src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -250,6 +250,13 @@ Copyright (c) .NET Foundation. All rights reserved.
       <RuntimePackAsset Remove="@(_ConflictPackageFiles)" Condition="'%(_ConflictPackageFiles.ConflictItemType)' != 'Reference'" />
     </ItemGroup>
 
+    <!-- Set valid RID platforms for runtime assets if targeting .NET 8+ and using the RID graph is not enabled. -->
+    <ItemGroup Condition="'$(_TargetFrameworkVersionWithoutV)' &gt;= '8.0'
```

Comment:
`&gt;=` looks like a typo, I don't think there's supposed to be a semicolon.  Or maybe I'm missing something about MSBuild syntax.

But either way, it's better to use `$([MSBuild]::VersionGreaterThanOrEquals($(_TargetFrameworkVersionWithoutV), '8.0'))` for this.

Reply:
The semicolon is intentional - part of the html character encoding. Will switch to the property function.